module/zfs/zfs_sa.c

```diff
@@ -433,6 +449,9 @@ zfs_sa_upgrade_txholds(dmu_tx_t *tx, znode_t *zp)
 	}
 }
 
+ZFS_MODULE_PARAM(zfs, zfs_, zil_saxattr_enabled, INT, ZMOD_RW,
+	"Enable xattr=sa extended attribute logging in zil.");
```

Comment:
This looks like a better description to use for the feature flag.

Reply:
Changed feature flag description to "Support for xattr=sa extended attribute logging in zil."